packages/remix-dev/cli/commands.ts

```diff
@@ -347,3 +348,45 @@ function purgeAppRequireCache(buildPath: string) {
     }
   }
 }
+
+export async function generateEntry(remixRoot: string, entry: string) {
+  // 0. valid entry file
+  let entries = new Set([
+    "entry.server.tsx",
+    "entry.server.js",
+    "entry.server.jsx",
+    "entry.client.tsx",
+    "entry.client.js",
+    "entry.client.jsx",
+  ]);
+  if (!entries.has(entry)) {
```

Comment:
`Array.includes` appeared to be faster

```suggestion
  if (!entries.includes(entry)) {
```